.eslintrc

```diff
@@ -29,7 +29,8 @@
     "prefer-template": 0,
     "import/no-unresolved": 0,
     "global-require": 0,
-    "no-underscore-dangle": 0
+    "no-underscore-dangle": 0,
+    "react/jsx-filename-extension": 0
```

Comment:
Maybe just [configure this rule](https://github.com/airbnb/javascript/issues/982#issuecomment-268397656) instead of complete disable?